src/snmalloc/backend/standard_range.h

```diff
@@ -0,0 +1,43 @@
+
+
+#pragma once
+
+#include "../backend/backend.h"
+
+namespace snmalloc
+{
+  template<
+    typename PAL,
+    typename Pagemap,
+    typename Base = EmptyRange,
+    size_t MinSizeBits = MinBaseSizeBits<PAL>()>
+  struct StandardLocalState
+  {
+    // Global range of memory, expose this so can be filled by init.
+    using GlobalR = Pipe<

```

Comment:
This looks as if it's the same as in the meta-protected one.  Can it be factored out into a common superclass?

Reply:
Not easily, I have just tried, but due to the way templates and inheritance work, I am writing considerably more code, and I don't think we are making it more readable.